### ISSUE TITLE
Bond.Compiler.CSharp doesn't include *_grpc.cs files 

### ISSUE THREAD

**Issue by eharshosh** [Microsoft/bond#448]
C# projects won't compile after adding the following directive to the .proj file:
```
<BondCodegen Include="myService.bond">
  <Options>--grpc</Options>
</BondCodegen>
```
If i'm not mistaken, Bond.Compiler.CSharp [includes *_types.cs file for compilation](https://github.com/Microsoft/bond/blob/87fa81d2f6f327895d6ed85d4de8c265a0b7370c/cs/build/nuget/Common.targets#L103), 
but not *_grpc.cs (after manually including myService_grpc.cs it compiles successfully)

Thanks in advance,
Eitan

**Comment by chwarr** [comment]
At the moment, this is the expected behavior. The targets were not updated to be taught about gRPC. Take a look at the [example C# projects](https://github.com/Microsoft/bond-grpc-examples/blob/0db9225e7773b1ad834ef29808f3670756cb409f/cs/calculator-client/calculator-client.csproj#L35-L45). They show how to edit your csproj file to get the build to work.

I'm going to leave this open with the above work around.

**Comment by eharshosh** [comment]
Thanks!

**Comment by hcoona** [comment]
I made #630 for it.

**Comment by chwarr** [comment]
Fixed in version 7.0.0
